src/components/OcSelect.vue

```diff
@@ -149,9 +160,6 @@ We can then retrieve all the values that we want to display from the slots param
         </span>
         <span class="option" v-text="desc" />
       </template>
-      <template #no-options>
```

Comment:
As it's still possible to include custom no options message via slot, keet this example pls.